demos/browser/app/meeting/meeting.ts

```diff
@@ -9,7 +9,7 @@ import {
   AudioVideoFacade,
   AudioVideoObserver,
   ClientMetricReport,
-  ConsoleLogger,
```

Comment:
Need to log ConsoleLogger by default and use POSTRequestLogger if a POST URL is supplied. Think about the case of someone checking out the repo for the first time and running the the meeting demo locally.